From bdba2945ef17362cffc282056153cb9d2ac8148f Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Tue, 3 May 2016 23:48:53 +0700
Subject: [PATCH 1/2] 
 0001-Rename-libjhead-libhead-not-to-confuse-our-camera-HA.patch

Change-Id: Ifcd514d25af2365d18690c6b182fc11d83be17c0
---
 Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 99a1766..fc987c9 100644
--- a/Android.mk
+++ b/Android.mk
@@ -28,7 +28,7 @@ LOCAL_SRC_FILES:= \
 	jpgfile.c \
 	makernote.c
 
-LOCAL_MODULE := libjhead
+LOCAL_MODULE := libhead
 
 LOCAL_SHARED_LIBRARIES := \
 	libcutils \
@@ -57,7 +57,7 @@ LOCAL_SHARED_LIBRARIES := \
 	libcutils \
 	libutils \
 	liblog \
-	libjhead
+	libhead
 
 include $(BUILD_SHARED_LIBRARY)
 
-- 
2.5.0

